bin/start_resin.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+systemctl start X.service
+systemctl start matchbox.service
+systemctl start screenly-viewer.service
+systemctl start screenly-web.service
+systemctl start screenly-websocket_server_layer.service
+
+mkdir /data/screenly /data/.screenly /data/screenly_assets /data/.config /data/.config/uzbl
+cp -n ansible/roles/screenly/files/screenly.conf /data/.screenly/screenly.conf
```

Comment:
We can't do this. This would overwrite the config on each start. Must be wrapped around some conditional logic (if file not exist..)

Reply:
Ah, ok. Sorry, overlooked that. I usually do an if condition for this.